### ISSUE TITLE
Open file in a new tab

### ISSUE THREAD

**Issue by tristone13th** [Vinzent03/obsidian-advanced-uri#118]
When opening a new file, it will replace the original file in the current tab, is there an option to open the URI in a new tab?

**Comment by Vinzent03** [comment]
You can use the `openmode` parameter https://vinzent03.github.io/obsidian-advanced-uri/concepts/navigation_parameters#open-mode

**Comment by tristone13th** [comment]
> You can use the `openmode` parameter [vinzent03.github.io/obsidian-advanced-uri/concepts/navigation_parameters#open-mode](https://vinzent03.github.io/obsidian-advanced-uri/concepts/navigation_parameters#open-mode)

Got it, many thanks!

**Comment by tristone13th** [comment]
@Vinzent03 During playing around openmode option, I found an issue when I'm using the following URI:

```
'obsidian://advanced-uri?vault=_posts&openmode=true&filepath=' + filepath + '&heading=' + heading
```

that it will open the file even the file is already opened, which is not consistent with the description in the documentation:

> true opens file in a new pane if not already opened

**Comment by Vinzent03** [comment]
You are right, I've fixed the documentation now and fixed more openmode issues. I hope it works now more like you expect.

**Comment by tristone13th** [comment]
> You are right, I've fixed the documentation now and fixed more openmode issues. I hope it works now more like you expect.

Thanks for your contribution! I think the original documentation is ok, what needs to be fixed is to make the behavior consistent with the documentation, that is, open the file in a new pane only if the file is not opened, otherwise just focus on the pane will be ok, what do you think? :-)

**Comment by Vinzent03** [comment]
Previously it was inconsistent with writing and just opening. Theoretically there are two different open mode needed. For when the the file is already opened and for when it's not. I'm currently unsure how to handle that.

**Comment by Vinzent03** [comment]
Okay I've changed it now again, so that `true` uses the existing pane if possible and if you it to always to open in a new tab use `tab`.

**Comment by tristone13th** [comment]
> Okay I've changed it now again, so that `true` uses the existing pane if possible and if you it to always to open in a new tab use `tab`.

Many thanks! I found in the latest release that if the file is already opened, then the `heading` parameter won't work and it simply focuses on the file without focusing on the specified heading.

**Comment by Vinzent03** [comment]
Should be fixed now :joy: Thanks for reporting all these issues!

**Comment by tristone13th** [comment]
> Should be fixed now 😂 Thanks for reporting all these issues!

Thanks for your great work!

**Comment by distbit0** [comment]
Hi wrt "You can set a default value in the plugin's settings. The value from the setting gets overwritten by specifying it in the URI." from the documentation, how can this setting be configured? I can not see any such option in the obsidian advanced URI plugin settings. many thx! :)

**Comment by tristone13th** [comment]
The `openmode=true` doesn't work again, @Vinzent03 could you check what's going wrong? Thx!

**Comment by Vinzent03** [comment]
@tristone13th What exactly doesn't work? Please share your uri and a minimal setup of your tabs. Works fine for me in a quick test.

**Comment by tristone13th** [comment]
I use this URI: `obsidian://advanced-uri?vault=tristone13th.github.io&openmode=true&filepath=Virtualization%2F2023-07-22-EPT.md&heading=EPT%20PTE%20bits%20illustrated%20%2F%20EPT%20format` to open a tab in my local vault but each time I visit this URI, it will always open a new tab even if there is an existing one opened.

**Comment by Vinzent03** [comment]
Sorry for the late reply, but for me, it's working fine. Are you on the latest version v1.42.0 ?
